Neos.ContentRepository/src/Service/ContentRepositoryBootstrapper.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Service;
+
+use Neos\ContentRepository\ContentRepository;
+use Neos\ContentRepository\Factory\ContentRepositoryServiceInterface;
+use Neos\ContentRepository\Feature\RootNodeCreation\Command\CreateRootNodeAggregateWithNode;
+use Neos\ContentRepository\Feature\WorkspaceCreation\Command\CreateRootWorkspace;
+use Neos\ContentRepository\Projection\Workspace\Workspace;
+use Neos\ContentRepository\SharedModel\Node\NodeAggregateIdentifier;
+use Neos\ContentRepository\SharedModel\NodeType\NodeTypeName;
+use Neos\ContentRepository\SharedModel\User\UserIdentifier;
+use Neos\ContentRepository\SharedModel\Workspace\ContentStreamIdentifier;
+use Neos\ContentRepository\SharedModel\Workspace\WorkspaceDescription;
+use Neos\ContentRepository\SharedModel\Workspace\WorkspaceName;
+use Neos\ContentRepository\SharedModel\Workspace\WorkspaceTitle;
+
+class ContentRepositoryBootstrapper implements ContentRepositoryServiceInterface
```

Comment:
we do not need to implement ContentRepositoryServiceInterface here.

A small code comment would be nice; and @api annotation.

Reply:
Good point, adjusted!